.github/workflows/security-secrets.yml

```
@@ -41,22 +42,23 @@ jobs:
           token: "${{ secrets.BOT_TOKEN || github.token }}"
 
       - name: Create SSH key
+        if: secrets.BOT_GIT_SSH_KEY != '' && secrets.GIT_USER_NAME != '' && secrets.GIT_USER_EMAIL != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)